data/tools/nginx-amplify.json

```diff
@@ -0,0 +1,21 @@
+[
+  {
+    "slug": "monitoring",
+    "name": "NGINX Amplify",
+    "description": "Amplify is free monitoring tool for NGINX. Amplify offers extended NGINX metric collection, visualization, monitoring, and a static analyzer for NGINX configuration. It's simple to set up, yet powerful enough to bring the much needed insight into NGINX performance. Amplify is a SaaS developed and operated by Nginx, Inc. Keep NGINX up to date and secure with a native monitoring solution.",
+    "url": "https://amplify.nginx.com",
+    "tags": [
+      "linux",
+      "bsd",
+      "free",
+      "commercial",
+      "ci", "cd",
+      "virt",
+      "cloud", "cloud-paas", "paas",
+      "logging", "monitoring",
```

Comment:
@rdsubhas I'm Product Marketing Manager in Amplify. Apologies, the landing page is due to be updated very soon and we are going out of beta shortly. We already provide stable service to our customers. Beta status means only marketing/financial issues, not a product quality.

I thought "all relevant tags apply" (as in "monitoring virt servers", or "using static analyzer for ci/cd"). Anyway, you can change the list of tags (remove redundant ones).

URL (amplify.nginx.com) is only one thing which I ask to keep – we don't like to use nginx.com/amplify in any external places like your list.

Also, you can read our grateful documentation which located here: https://amplify.nginx.com/docs/